[2024-05-19T20:26:49.869+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-19T20:26:49.896+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-05-14T00:00:00+00:00 [queued]>
[2024-05-19T20:26:49.909+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-05-14T00:00:00+00:00 [queued]>
[2024-05-19T20:26:49.912+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-19T20:26:49.985+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): process_parquet_files> on 2023-05-14 00:00:00+00:00
[2024-05-19T20:26:49.997+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=745) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-19T20:26:50.000+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'silver_s3', 'process_parquet_files', 'scheduled__2023-05-14T00:00:00+00:00', '--job-id', '263', '--raw', '--subdir', 'DAGS_FOLDER/silver_s3.py', '--cfg-path', '/tmp/tmp9ujfo717']
[2024-05-19T20:26:50.004+0000] {standard_task_runner.py:91} INFO - Job 263: Subtask process_parquet_files
[2024-05-19T20:26:50.003+0000] {standard_task_runner.py:63} INFO - Started process 770 to run task
[2024-05-19T20:26:50.148+0000] {task_command.py:426} INFO - Running <TaskInstance: silver_s3.process_parquet_files scheduled__2023-05-14T00:00:00+00:00 [running]> on host 620090466bc8
[2024-05-19T20:26:50.343+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='silver_s3' AIRFLOW_CTX_TASK_ID='process_parquet_files' AIRFLOW_CTX_EXECUTION_DATE='2023-05-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-14T00:00:00+00:00'
[2024-05-19T20:26:50.349+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-19T20:26:50.407+0000] {base.py:84} INFO - Using connection ID 'aws_s3' for task execution.
[2024-05-19T20:26:50.408+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_s3', conn_type='aws') credentials retrieved from login and password.
[2024-05-19T20:26:55.428+0000] {logging_mixin.py:188} INFO - No .parquet files found.
[2024-05-19T20:26:55.431+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-19T20:26:55.433+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-19T20:26:55.442+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=silver_s3, task_id=process_parquet_files, run_id=scheduled__2023-05-14T00:00:00+00:00, execution_date=20230514T000000, start_date=20240519T202649, end_date=20240519T202655
[2024-05-19T20:26:55.461+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-19T20:26:55.478+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-19T20:26:55.479+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-05-19T20:36:37.667+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-19T20:36:37.676+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-05-14T00:00:00+00:00 [queued]>
[2024-05-19T20:36:37.681+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-05-14T00:00:00+00:00 [queued]>
[2024-05-19T20:36:37.681+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-19T20:36:37.688+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): process_parquet_files> on 2023-05-14 00:00:00+00:00
[2024-05-19T20:36:37.694+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1877) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-19T20:36:37.696+0000] {standard_task_runner.py:63} INFO - Started process 1885 to run task
[2024-05-19T20:36:37.699+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'silver_s3', 'process_parquet_files', 'scheduled__2023-05-14T00:00:00+00:00', '--job-id', '376', '--raw', '--subdir', 'DAGS_FOLDER/silver_s3.py', '--cfg-path', '/tmp/tmpk0wbrcoj']
[2024-05-19T20:36:37.710+0000] {standard_task_runner.py:91} INFO - Job 376: Subtask process_parquet_files
[2024-05-19T20:36:37.755+0000] {task_command.py:426} INFO - Running <TaskInstance: silver_s3.process_parquet_files scheduled__2023-05-14T00:00:00+00:00 [running]> on host 620090466bc8
[2024-05-19T20:36:37.810+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='silver_s3' AIRFLOW_CTX_TASK_ID='process_parquet_files' AIRFLOW_CTX_EXECUTION_DATE='2023-05-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-14T00:00:00+00:00'
[2024-05-19T20:36:37.811+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-19T20:36:37.822+0000] {base.py:84} INFO - Using connection ID 'aws_s3' for task execution.
[2024-05-19T20:36:37.822+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_s3', conn_type='aws') credentials retrieved from login and password.
[2024-05-19T20:36:45.660+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-19T20:36:45.663+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/silver_s3.py", line 69, in process_parquet_files
    os.remove(file_path)
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/disney_data_20240518T000000.parquet'
[2024-05-19T20:36:45.675+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=silver_s3, task_id=process_parquet_files, run_id=scheduled__2023-05-14T00:00:00+00:00, execution_date=20230514T000000, start_date=20240519T203637, end_date=20240519T203645
[2024-05-19T20:36:45.691+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 376 for task process_parquet_files ([Errno 2] No such file or directory: '/tmp/disney_data_20240518T000000.parquet'; 1885)
[2024-05-19T20:36:45.729+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-05-19T20:36:45.742+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-19T20:36:45.743+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
